app/services/edge_cache/bust_podcast.rb

```diff
@@ -0,0 +1,9 @@
+module EdgeCache
+  class BustPodcast < Bust
+    def self.call(path)
+      return unless path
+
+      bust("/#{path}")
+    end
+  end
+end
```

Comment:
Technically, we don't _really_ need this service since it just busts the path itself. I thought it would be nice to leave as a dedicated service though because it makes it much easier to handle Podcast specific caches in the future.

Reply:
This makes sense. I could see this having more functionality in the future, so keeping the interface seems good.

Reply:
I like that you pulled it into a service -- even if it's simple for now, I prefer the consistency 👍🏽